lenskit-core/src/main/java/org/lenskit/data/ratings/Rating.java

```diff
@@ -143,6 +169,51 @@ public double getValue() {
         return value;
     }
 
+    @Override
```

Comment:
If this is what view objects look like its probably going to make people (who have to implement views) sad. a lot of "mandatory repeat yourself" and strange template code most people won't understand. I don't know how to fix this problem.